packages/workflow/src/constants.ts

```diff
@@ -97,6 +98,7 @@ export const RESPOND_TO_CHAT_NODE_TYPE = '@n8n/n8n-nodes-langchain.chat';
 export const MEMORY_MANAGER_NODE_TYPE = '@n8n/n8n-nodes-langchain.memoryManager';
 export const MEMORY_BUFFER_WINDOW_NODE_TYPE = '@n8n/n8n-nodes-langchain.memoryBufferWindow';
 export const GUARDRAILS_NODE_TYPE = '@n8n/n8n-nodes-langchain.guardrails';
+export const SEAR_XNG_TOOL_NODE_TYPE = '@n8n/n8n-nodes-langchain.toolSearXng';
```

Comment:
If the plan is to add all supported tools, it will be messy adding all of them here. Maybe worth considering refactoring to import from node package. 

Community and custom nodes then could be supported as tools in the chat.

Reply:
happy to leave as post p1 refactor